egs/europarl_st/SRT/lcma_srt/decode.py

```
@@ -0,0 +1,1538 @@
+            )
+            _accumulate_moe_stats(
+                moe_stats_st,
+                moe_counts_st,
+                batch_moe_stats.get("st"),
+            )
+
+        if params.asr_decode:
+            for name, hyps in asr_hyps_dict.items():
+                this_batch = []
+                assert len(hyps) == len(texts_asr)
+                for cut_id, hyp_words, ref_text in zip(cut_ids, hyps, texts_asr):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)